src/services/license.types.ts

```diff
@@ -194,13 +194,17 @@ export type LicenseCreateRequest = Omit<
 export type LicenseUpdateRequest = Omit<
   LicenseResponse,
   | LicenseReadOnlyProperties
+  | "productId"
   | "key"
   | "licensePolicyId"
   | "createdAt"
-  | "type"
   | "validity"
   | "metadata"
```

Comment:
metadata and meterattributes can be added updated

Reply:
Since the `Omit` type is working with `LicenseResponse` as a base, for `LicenseUpdateRequest` and similar, we omit metadata and meterAttributes and add them again based on the `MetadataRequest` type.